servicetest/testframework/lib.py

```diff
@@ -118,50 +118,51 @@ def start(self, data):
 
             The values in the dict are passed as arguments to the D-Bus methods on SoftwareContainerAgent.
             The dict is mapped like so:
-            Container.CONFIG - argument to SoftwareContainerAgent::CreateContainer
-            Container.HOST_PATH - second argument to SoftwareContainerAgent::BindMountFolderInContainer
-            Container.BIND_MOUNT_DIR - third argument to SoftwareContainerAgent::BindMountFolderInContainer
-            Container.READONLY - fourth argument to SoftwareContainerAgent::BindMountFolderInContainer
+            Container.CONFIG - argument to SoftwareContainerAgent::Create
+            Container.HOST_PATH - second argument to SoftwareContainerAgent::BindMount
+            Container.BIND_MOUNT_DIR - third argument to SoftwareContainerAgent::BindMount
+            Container.READONLY - fourth argument to SoftwareContainerAgent::BindMount
         """
         success = self.__create_container(data[Container.CONFIG])
         if False == success:
             print "Failed to create container"
             return False
         
-        self.__bind_dir, success = self.__bindmount_folder_in_container(data[Container.HOST_PATH],
-                                                               data[Container.BIND_MOUNT_DIR],
-                                                               data[Container.READONLY])
+        success = self.__bindmount_folder_in_container(data[Container.HOST_PATH],
+                                                       data[Container.BIND_MOUNT_DIR],
+                                                       data[Container.READONLY])
         if False == success:
             print "Failed to retrieve binded folder"
             return False
-
-        return True
+        else:
+            self.__bind_dir = data[Container.BIND_MOUNT_DIR]
```

Comment:
Since you need to provide an absolute path to which to mount, we don't have that echoed back by the agent, so if the mounting was successful it was indeed mounted to where we asked it to. That's why it is only happening if mounting was successful.